iconservice/icon_config.py

```diff
@@ -40,7 +39,7 @@
         ConfigKey.UN_STAKE_LOCK_MAX: IISS_DAY_BLOCK * 20
     },
     ConfigKey.IISS_CALCULATE_PERIOD: IISS_DAY_BLOCK,
```

Comment:
This field default value also changed to 43_120

Reply:
we can reduce tmp config setting variables using IISS_CALCULATE_PERIOD.
actually, and we must use 43200 before decentralize prep for supporting before version.

Reply:
I understood what you say. I recommend you add comments about this history